<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>WelcomeWidget</class>
  <widget class="QWidget" name="WelcomeWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>502</width>
        <height>360</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <item rowspan="1" row="1" column="0" colspan="2" >
        <spacer name="spacer1" >
          <property name="sizeHint" >
            <size>
              <width>31</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="1" column="3" colspan="2" >
        <spacer name="spacer1_2" >
          <property name="sizeHint" >
            <size>
              <width>31</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="2" >
        <widget class="QLabel" name="pix" >
          <property name="text" >
            <string>pix</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="2" >
        <spacer name="spacer5" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>21</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="2" column="2" >
        <spacer name="spacer6" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="3" column="0" colspan="5" >
        <widget class="QLabel" name="disclamer" >
          <property name="text" >
            <string>All software developed by TUBITAK/UEKAE is covered by GNU/GPL. Other software have their corresponding licenses. By installing or using Pardus, you agree to accept them. Do you want to continue?</string>
          </property>
          <property name="alignment" >
            <set>Qt::AlignTop</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="5" column="2" >
        <spacer name="spacer7" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="4" column="0" >
        <spacer name="spacer12_2" >
          <property name="sizeHint" >
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="4" column="4" >
        <spacer name="spacer12" >
          <property name="sizeHint" >
            <size>
              <width>30</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="4" column="1" colspan="3" >
        <layout class="QGridLayout" >
          <property name="spacing" >
            <number>10</number>
          </property>
          <item row="1" column="0" >
            <layout class="QHBoxLayout" >
              <property name="spacing" >
                <number>12</number>
              </property>
              <item>
                <widget class="QPushButton" name="gplButton" >
                  <property name="minimumSize" >
                    <size>
                      <width>130</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="text" >
                    <string>Read GPL</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="rebootButton" >
                  <property name="minimumSize" >
                    <size>
                      <width>130</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="text" >
                    <string>Reboot System!</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item row="0" column="0" >
            <widget class="Q3ButtonGroup" name="buttonGroup1" >
              <property name="maximumSize" >
                <size>
                  <width>32767</width>
                  <height>30</height>
                </size>
              </property>
              <property name="frameShape" >
                <enum>Q3GroupBox::NoFrame</enum>
              </property>
              <property name="title" >
                <string/>
              </property>
              <layout class="QGridLayout" >
                <item row="0" column="0" >
                  <widget class="QRadioButton" name="accept" >
                    <property name="text" >
                      <string>Accept!</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="QRadioButton" name="not_accept" >
                    <property name="text" >
                      <string>Don't Accept!</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>accept</tabstop>
    <tabstop>not_accept</tabstop>
    <tabstop>gplButton</tabstop>
    <tabstop>rebootButton</tabstop>
  </tabstops>
</ui>
